examples/server/webui/index.html

```diff
@@ -153,7 +153,7 @@ <h2 class="font-bold ml-4">Conversations</h2>
             :disabled="isGenerating"
             id="msg-input"
           ></textarea>
-          <button v-if="!isGenerating" class="btn btn-primary ml-2" @click="sendMessage" :disabled="inputMsg.length === 0">Send</button>
+          <button v-if="!isGenerating" class="btn btn-primary ml-2" @click="sendMessage" :disabled="inputMsg.length === 0" id="msg-send">Send</button>
```

Comment:
This button calls `sendMessage()` function as stated in `@click`. You should call the underlay function instead of assigning an ID for it.

Reply:
Thanks, will give that a try.

Reply:
I got the method call to work with (abbreviated):
```
  const appInstance = mainApp.mount('#app');
  appInstance.sendMessage();
```

However, the focus() call further down still needs the id:
```
      setTimeout(() => document.getElementById('msg-input').focus(), 1);
```

Do you have a better idea to reliably focus the button DOM element without an id?